src/about.html

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
```

Comment:
@juan-ortizc single-quotes are only used in JS files, for HTML files use double-quotes. Please revert the changes on this file

Reply:
resolved